tests/rptest/services/redpanda.py

```diff
@@ -83,11 +84,17 @@
 
 
 class BadLogLines(Exception):
-    def __init__(self, nodes):
-        self.nodes = nodes
+    def __init__(self, node_to_lines):
+        self.node_to_lines = node_to_lines
```

Comment:
Just trying to parse types here.. a place where adding type annotations improves readabililty. IIUC, type of `node_to_lines` is `dict[str, list[str]]`, right?

Reply:
Ahh.. I see below that it is more like `dict[ClusterNode, list[str]]`?

Reply:
right.